examples/usecases/aci_mcp/aci_toolkit_camel.py

```diff
@@ -0,0 +1,74 @@
+# ========= Copyright 2023-2024 @ CAMEL-AI.org. All Rights Reserved. =========
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ========= Copyright 2023-2024 @ CAMEL-AI.org. All Rights Reserved. =========
+#!/usr/bin/env python3
+import os
+
+from dotenv import load_dotenv
+from rich import print as rprint
+
+from camel.agents import ChatAgent
+from camel.models import ModelFactory
+from camel.toolkits import ACIToolkit
+from camel.types import ModelPlatformType, ModelType
+
+load_dotenv()
+
+
+def main():
+    rprint("[green]CAMEL AI with ACI Toolkit[/green]")
+
+    # get the linked account from env or use default
+    linked_account = os.getenv("LINKED_ACCOUNT_OWNER", "parthshr370")
+    rprint(f"Using account: [cyan]{linked_account}[/cyan]")
+
+    # setup aci toolkit
+    aci_toolkit = ACIToolkit(linked_account_owner_id=linked_account)
+    tools = aci_toolkit.get_tools()
+    rprint(f"Loaded [cyan]{len(tools)}[/cyan] tools")
+
+    # setup gemini model
+    model = ModelFactory.create(
+        model_platform=ModelPlatformType.GEMINI,
+        model_type=ModelType.GEMINI_2_5_PRO_PREVIEW,
+        api_key=os.getenv("GOOGLE_API_KEY"),
```

Comment:
Let's name it as `GEMINI_API_KEY`. `GOOGLE_API_KEY` usually is for google search